Binary files NAMD_2.14_Source.orig/.git/index and NAMD_2.14_Source/.git/index differ
diff -Nru NAMD_2.14_Source.orig/src/Molecule.C NAMD_2.14_Source/src/Molecule.C
--- NAMD_2.14_Source.orig/src/Molecule.C	2022-04-04 12:32:49.000000000 +0200
+++ NAMD_2.14_Source/src/Molecule.C	2022-04-28 18:45:08.000000000 +0200
@@ -10159,7 +10159,7 @@
     numLJsites1 = numLJsites2 = numLJsites;
     int alch_counter = 0;
     for (int i=0; i < numAtoms; ++i) {
-      int alchFlagi;
+      int alchFlagi = 0;
       if (get_fep_type(i) == 2 || get_fep_type(i) == 4) alchFlagi = -1;
       if (get_fep_type(i) == 1 || get_fep_type(i) == 3) alchFlagi = 1;
       if (params->get_vdw_pair_params(atoms[i].vdw_type, atoms[i].vdw_type,
@@ -10182,7 +10182,7 @@
         else if (alchFlagi == -1) numLJsites1--;
       }
       for (int j=i+1; j < numAtoms; ++j) {
-        int alchFlagj;
+        int alchFlagj = 0;
         if (get_fep_type(j) == 2 || get_fep_type(j) == 4) alchFlagj = -1;
         if (get_fep_type(j) == 1 || get_fep_type(j) == 3) alchFlagj = 1;
         int alchFlagSum = alchFlagi + alchFlagj;
diff -Nru NAMD_2.14_Source.orig/src/ScriptTcl.C NAMD_2.14_Source/src/ScriptTcl.C
--- NAMD_2.14_Source.orig/src/ScriptTcl.C	2022-04-04 12:32:49.000000000 +0200
+++ NAMD_2.14_Source/src/ScriptTcl.C	2022-04-13 11:14:09.000000000 +0200
@@ -2085,11 +2085,11 @@
 }
 
 
-extern "C" void newhandle_msg(void *v, const char *msg) {
+extern "C" void newhandle_msg(void *vdata, void *v, const char *msg) {
   CkPrintf("psfgen) %s\n",msg);
 }
 
-extern "C" void newhandle_msg_ex(void *v, const char *msg, int prepend, int newline) {
+extern "C" void newhandle_msg_ex(void *vdata, void *v, const char *msg, int prepend, int newline) {
   CkPrintf("%s%s%s", (prepend ? "psfgen) " : ""), msg, (newline ? "\n" : ""));
 }
 
